test/cpp/interop/xds_interop_client.cc

```diff
@@ -265,9 +291,11 @@ class TestClient {
     }
     std::chrono::system_clock::time_point deadline =
         std::chrono::system_clock::now() +
-        std::chrono::seconds(absl::GetFlag(FLAGS_rpc_timeout_sec));
+        std::chrono::seconds(config.timeout_sec != 0
```

Comment:
Since this PR includes the change for the proto, which includes the change to `ClientConfigureRequest` by adding the `timeout_sec` field. What about adding the timeout value to the exiting usage of `ClientConfigureRequest` in the circuit breaking test driver? That is, we will always respect the `timeout_sec` from `RpcConfig`. Then we can eliminate the two special handling for timeout_sec here and [line 240](https://github.com/grpc/grpc/blob/2cc1abde097e49699887a79f59e1fed767d56f36/test/cpp/interop/xds_interop_client.cc#L240).

Reply:
Well, [Go](https://github.com/grpc/grpc-go/blob/7f2581f910fc21497091c4109b56d310276fc943/interop/xds/client/client.go#L390) is still using the command line option rpcTimeout as the default. But I guess one major reason is because existing uses in the test driver does not set it. So anyway, the test driver needs to make the change first before client implementations can do it.